File: src/DashSpec.Core/Parsing/Lexing/TokenKind.cs
namespace DashSpec.Core.Parsing;

internal enum TokenKind
{
    At,
    Bang,
    LBrace,
    RBrace,
    Eq,
    DotDot,
    RelativeDay,
    Comma,
    LBracket,
    RBracket,
    LParen,
    RParen,
    Ident,
    String,
    Raw,
    Newline,
    Eof,
}

File: src/DashSpec.Modeling.Parse/Lexing/TokenKind.fs
namespace DashSpec.Modeling.Parse.Lexing

type TokenKind =
    | At
    | Bang
    | LBrace
    | RBrace
    | Eq
    | DotDot
    | Dot
    | Slash
    | RelativeDay
    | Comma
    | LBracket
    | RBracket
    | LParen
    | RParen
    | Ident
    | String
    | HexColor
    | Raw
    | LineComment
    | BlockComment
    | Newline
    | Eof

